composites/Plugin/SnippetPreview/components/SnippetPreview.js

```diff
@@ -605,7 +613,8 @@ export default class SnippetPreview extends PureComponent {
 						             onClick={ onClick.bind( null, "description" ) }
 						             onMouseOver={ partial( onMouseOver, "description" ) }
 						             onMouseLeave={ partial( onMouseLeave, "description" ) }
-						             innerRef={ this.setDescriptionRef } >
+						             innerRef={ this.setDescriptionRef }
+						             overflowHidden={ this.state.descriptionFits }>
```

Comment:
See above, I'd ex[ect the other way around: overflowHidden true when description does _not_ fit.